setup.cfg

```diff
@@ -1,12 +1,105 @@
+[metadata]
+name = ctapipe
+description = CTA Python pipeline experimental version
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = ctapipe developers
+author_email = karl.kosack@cea.fr
+license = BSD 3-Clause License
+url = https://github.com/cta-observatory/ctapipe
+classfisers =
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Astronomy
+    Development Status :: 3 - Alpha
+
+
+[options]
+packages = find:
+python_requires = >=3.8
+zip_safe = False
+install_requires=
+	astropy~=5.0
+	bokeh~=2.0
+	docutils
+	eventio>=1.9.1<2.0.0a0
+	iminuit>=2
+	importlib_resources ; python_version < "3.9"
+	joblib
+	matplotlib~=3.0
+	numba>=0.43
+	numpy~=1.16
+	psutil
+	pyyaml>=5.1
+	requests
+	scikit-learn
+	scipy~=1.2
+	tables~=3.4
+	tqdm>=4.32
+	traitlets~=5.0,>=5.0.5
+	zstandard
+
+
+[options.extras_require]
+tests =
+    pytest
+    pandas>=0.24.0
+    tomli
+    pytest_astropy_header
+	h5py
+
+
+docs =
+    sphinx~=3.5
+	jinja2~=3.0.2  # for sphinx 3.5, remove when upgrading sphinx
```

Comment:
can this comment be translated into a dependency, as in line 32 for `imporlib_resources`?

Reply:
I don't think you can make this dependent on arbitrary versions, only on python versions